packages/postcss-reduce-idents/src/lib/grid-template.js

```diff
@@ -41,7 +41,7 @@ module.exports = function () {
         });
 
         declCache.push(node);
-      } else if (node.prop.toLowerCase() === 'grid-area') {
+      } else if (/^(grid-area|grid-column|grid-row)/i.test(node.prop)) {
```

Comment:
I think this includes the `-start` and `-end` properties as well. Wouldn't it be better to list the properties explicitly instead of relying on a regular expression? That way there are no surprises if a new property is introduced or if someone uses non-standard syntax. Something like

```js
const gridProperties = new Set(['grid-somethingg'....']);
if (gridProperties.has(node.prop.toLowerCase()) {
```
Also avoids recreating a RegEx object (although the performance gain is likely undetectable).